ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -73,6 +81,13 @@ class AlertsView extends React.Component {
     }
   }
 
+  extendDropdownOptions = frameworks => {
+    const frameworkOptions = cloneDeep(frameworks);
```

Comment:
Doesn't `const frameworkOptions = [...frameworks]` achieve the same thing?

Reply:
No, since in JS the syntax with `...` does a shallow copy. Without `cloneDeep` the changes would reflect into the GraphView and CompareView components and this is not wanted.

Reply:
That's true, but only for nested objects and arrays.

Reply:
Just printed `frameworks` and it is a list of objects, like this: `const frameworks = [  { id: -1, name: 'all' },...]`. List of objects.

Reply:
oh, right. My bad :)